tests/integration/test_features.py

```diff
@@ -144,15 +145,14 @@ def _deploy_app(temp_dirname):
         chalice_stage_name='dev',
         autogen_policy=True
     )
-    d = factory.create_default_deployer(
-        factory.create_botocore_session(), None)
+    session = factory.create_botocore_session()
+    d = factory.create_new_default_deployer(
```

Comment:
Any reason we're creating a session twice?

Reply:
Nope, I'll fix that before merging.